cypress/support/pages/course/pageShareCourseModal.js

```diff
@@ -1,12 +1,16 @@
-"use strict"
+"use strict";
 
```

Comment:
not sure if its a good idea to separate each modal files, in that way same selector are being used in multiple files and maintenance is not that easy for them